openapi/openapi.yaml

```diff
@@ -931,8 +930,212 @@ paths:
+          required: false
+          description: Page offset in pagination.
+          schema:
+            type: integer
+            default: 0
+      responses:
+        "200":
+          description: A list of build analyzer results available.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/AnalysisListingResponse"
+        "400":
+          description: On invalid request.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/AnalysisResponseError"
+
+  /buildlog-analyse/{analysis_id}:
+    get:
+      tags: ["Buildlogs"]
+      x-openapi-router-controller: thoth.user_api.api_v1
+      operationId: get_buildlog_analyze
+      summary: Retrieve a build analyzer result.
+      parameters:
+        - name: analysis_id
+          in: path
+          required: true
+          description: Id of analysis that results should be retrieved.
+          schema:
+            type: string
+      responses:
+        "200":
+          description: Build Analyzer result retrieved.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/AnalysisResultResponse"
+        "202":
+          description: Results are not ready yet.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/AnalysisUnfinishedResultResponse"
+        "400":
+          description: On invalid request.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/AnalysisResponseError"
+        "404":
+          description: The given document does not exist.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/AnalysisResponseError"
+
 components:
   schemas:
+    Build:
+      type: object
+      properties:
+        base_image:
+          type: string
+          description: >
+            Name of base image used by the s2i build.
+          example: >
+            registry.centos.org/centos/python-36-centos7
+        output_image:
+          type: string
+          description: >
+            Name of output image - can also specify remote registry to pull
+            image from.
+          example: >
+            registry.centos.org/centos/python-36-centos7
+        log:
+          type: string
+          description: >
+            A full build or installation log that was output during image build.
+          example: >
+            Collecting tensorflow
+            Downloading tensorflow-1.4.1-cp36-cp36m-manylinux1_x86_64.whl (41.2MB)
+            Successfully installed tensorflow-1.4.1
+        apiversion:
+          type: string
+          description: >
+            BuildLog api version.
+          example: >
+            "apis/build.openshift.io/v1/namespaces/$NAMESPACE/builds HTTP/1.1"
```

Comment:
It might be a good idea to create `build_log` which will nest `log` itself` `apiVersion`, `kind` and other entries from OpenShift event as they semantically correspond to each other on that level.